docs/quick-start-guide.md

```diff
@@ -97,3 +98,35 @@ At the moment we have:
 * [Arch](https://aur.archlinux.org/packages/mgmt/)
 
 Please contribute more! We'd especially like to see a Debian package!
+
+## OSX/macOS/Darwin development
+Developing and running `mgmt` on macOS is currently not supported (but not discouraged either). Meaning it might work but in the case it doesn't you would have to provide your own patches to fix problems (the project maintainer and community are glad to assist where needed).
+
+There are currently some issues that make `mgmt` less suitable to run for provisioning macOS (eg: https://github.com/purpleidea/mgmt/issues/33). But as a client to provision remote servers it should run fine.
+
+Since the primary supported systems are Linux and these are the environments tested for it is wise to run these suites during macOS development as well. To ease this Docker can be levaraged ((Docker for Mac)[https://docs.docker.com/docker-for-mac/]).
+
+Before running any of the commands below create the development Docker image:
+
+    docker/scripts/build-development
+
+This image requires updating every time dependencies (`make-deps.sh`) change.
+
+Then to run the test suite:
+
+    docker run --rm -ti \
+      -v $PWD:/go/src/github.com/purpleidea/mgmt/ \
```

Comment:
Does it make sense to remove all the space identation and surround this with three backticks on either side?

Reply:
I'll stick with triple quotes as that seems to be the convention for this project.